src/services/lnd/utils.ts

```diff
@@ -53,16 +50,15 @@ export const deleteExpiredInvoiceUser = async () => {
 }
 
 export const deleteFailedPaymentsAllLnds = async () => {
-  baseLogger.warn("only run deleteFailedPayments on lnd 0.13")
-  return Promise.resolve()
-  // try {
-  //   const lnds = offchainLnds
-  //   for (const { lnd } of lnds)
-  //     await deleteFailedPayments({lnd})
-  //   }
-  // } catch (err) {
-  //   baseLogger.warn({ err }, "error deleting failed payment")
-  // }
+  const lnds = offchainLnds
+  for (const { lnd, socket } of lnds) {
+    try {
+      baseLogger.info({ socket }, "running deleteFailedPayments")
+      await deleteFailedPayAttempts({ lnd })
```

Comment:
do we want to delete only payment attemps or failed payments too? https://github.com/alexbosworth/ln-service#deletefailedpayments

Reply:
I have the same question as @dolcalmi. If we're only deleting failed payment attempts then we should also rename the function and change the log message accordingly.

Reply:
@dolcalmi eventually we want to delete all failed payment, but starting with the payment attempt is less risky.

I'm not sure yet if there would be some side effect of deleting failed payment for the refund. but deleting payment attempt should have no effect

I'll do the renaming